src/EditorFeatures/VisualBasicTest/CodeActions/EncapsulateField/EncapsulateFieldTests.vb

```diff
@@ -305,7 +307,7 @@ Class C
             Return [Class]
         End Get
         Set(value As String)
-            Me.Class = value
+            [Class] = value
```

Comment:
Could you add another test that performs this particular escaping scenario with `QualifyFieldAccess` set to true?